IdentityCore/src/requests/broker/MSIDSSOExtensionTokenRequestDelegate.m

```diff
@@ -31,19 +31,22 @@ @implementation MSIDSSOExtensionTokenRequestDelegate
 
 - (void)authorizationController:(__unused ASAuthorizationController *)controller didCompleteWithAuthorization:(ASAuthorization *)authorization
 {
+    MSID_LOG_WITH_CTX_PII(MSIDLogLevelInfo, self.context, @"Received response from SSO extension with authorization is nil: %@", authorization ? @"No" : @"Yes");
```

Comment:
Can you elaborate why we need PII here?

Reply:
got catch, I will change to MSID_LOG_WITH_CTX